docs/SUMMARY.md

```diff
@@ -82,6 +82,7 @@
   - [Release Cycle](development/releases.md)
   - [Git Usage](development/git.md)
   - [Testing]()
+    - [Demo scripts](development/demo.md)
```

Comment:
Heads up that this should be under `development/testing/demo.md` according to the [doc docs](https://github.com/matrix-org/synapse/tree/develop/docs#adding-to-the-documentation).

The fact many things in this file *don't* yet follow that convention is tracked at #11274.